Assets/css/style.css

```diff
@@ -0,0 +1,33 @@
+/* Navigation Bar Styles */
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  width: 200px;
+  background-color: #f1f1f1;
+}
+
+li a {
```

Comment:
Please replace "li a" with "nav ul li a" just so it doesnt effect all LI's on the page